DepositeCalcTests/Tests/RegisterPageTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using DepositeCalcTests.Pages;
+using DepositeCalcTests.Utilities;
+using NUnit.Framework;
+
+namespace DepositeCalcTests.Tests
+{
+    internal class RegisterPageTests : BaseTest
+    {
+        private RegisterPage registerPage;
+        private ApiHelper apiHelper;
+
+        [SetUp]
+        public void Setup()
+        {
+            InitDriver("https://localhost:5001/Register");
+            registerPage = new RegisterPage(driver);
+            AssertPageTitle("Register");
+            apiHelper = new ApiHelper();
+            apiHelper.Delete("tomcruise");
```

Comment:
If you make the methods 'static' you don't need to create an instance of the class to call this method.

`Sorry, that was my mistake.`
```suggestion
            ApiHelper.Delete("tomcruise");
```